docs/molgenis/dev_beaconv2.md

```diff
@@ -0,0 +1,91 @@
+
+Framework endpoints for which GET requests are implemented are:
+- `/`, `/info` and `/service-info` return Beacon metadata described in the GA4GH Service Info schema.
+- `/configuration` returns configuration aspects and the definition of the entry types.
+- `/map` returns a map (like a web sitemap) of the different endpoints implemented in this Beacon instance.
+- `/entry_types` returns the section of the configuration that describes the entry types in this Beacon.
+
+Model endpoints for which record-level GET requests are implemented are:
+- `/analyses` for entry type [Analyses](https://docs.genomebeacons.org/schemas-md/analyses_defaultSchema/). Data is retrieved from the _Analyses_ table.
+- `/biosamples` for entry type [Biosamples](https://docs.genomebeacons.org/schemas-md/biosamples_defaultSchema/). Data is retrieved from the _Biosamples_ table.
+- `/cohorts` for entry type [Cohorts](https://docs.genomebeacons.org/schemas-md/cohorts_defaultSchema/). Data is retrieved from the _Cohorts_ table.
+- `/datasets` for entry type [Datasets](https://docs.genomebeacons.org/schemas-md/datasets_defaultSchema/). Returns the names and timestamps of available database schemas.
+- `/g_variants` for entry type [Genomic Variations](https://docs.genomebeacons.org/schemas-md/genomicVariations_defaultSchema/). Data is retrieved from the _GenomicVariations_ table.
+- `/runs` for entry type [Runs](https://docs.genomebeacons.org/schemas-md/runs_defaultSchema/). Data is retrieved from the _Runs_ table.
+- `/individuals` for entry type [Individuals](https://docs.genomebeacons.org/schemas-md/individuals_defaultSchema/). Data is retrieved from the _Individuals_ table.
+
+Endpoints that are not yet implemented:
+- `/filtering_terms` returns a list of the filtering terms accepted by that Beacon instance.
+
+### Queries
+The Beacon v2 framework allows for a range of potential queries and filters.
+The following ones are currently implemented.
+
+#### ID queries
+Simple identifier-based queries are accepted via GET on the endpoints for Analyses, Biosamples, Cohorts, Individuals and Runs.
+For instance:
+- `<server>/api/beacon/analyses?id=A01`
+- `<server>/api/beacon/biosamples?id=Sample0001`
+- `<server>/api/beacon/cohorts?id=Cohort0001`
+- `<server>/api/beacon/individuals?id=Ind001`
+- `<server>/api/beacon/runs?id=SRR10903401`
+
+#### Count queries
+Count queries are accepted as POST requests on the `/individuals` endpoint.
+These requests should conform to specifications available [here](https://github.com/ejp-rd-vp/vp-api-specs).
+The response contains the number of individuals that match the supplied filters.
+
+#### Genomic queries
+On genomic variation, a number of different genomic queries are accepted via GET on the `/g_variants` endpoint.
+- Sequence query, [example](https://vkgl-emx2.molgeniscloud.org/api/beacon/g_variants?start=32936732&referenceName=13&referenceBases=G&alternateBases=C)
```

Comment:
this is a production server?

Reply:
Yes, it has production status so this example should survive